crates/circuit/src/parameter/symbol_expr.rs

```diff
@@ -905,7 +900,7 @@ impl SymbolExpr {
     }
 
     pub fn string_id(&self) -> String {
```

Comment:
I'm still not a fan of calling this an "id" since it sounds like what gets returned is a simple ID of a constant length.